pkg/cmd/render/render.go

```diff
@@ -188,13 +181,7 @@ func render() error {
 	// render operator config
 	var operatorConfig bytes.Buffer
 	templateVars := operatorTemplateVars{
-		ForceCredentialsMode: installConfigMode,
-	}
-
-	// if either the legacy config map or the install-config have CCO as disabled/manual,
-	// then we must set the config appropriately.
-	if operatorDisabled {
-		templateVars.ForceCredentialsMode = string(operatorv1.CloudCredentialsModeManual)
+		CredentialsMode: string(effectiveMode),
```

Comment:
Does the template work when `CredentialsMode` has the type `operatorv1.CloudCredentialsMode` instead of `string`?

Reply:
It certainly does.